block.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "create-block/review-block",
```

Comment:
Rename this to `sudar/wp-plugin-review-block`

Reply:
Fixed.